tests/shell/pipeline/commands/external.rs

```diff
@@ -39,6 +39,25 @@ fn automatically_change_directory() {
     })
 }
 
+#[test]
+fn automatically_change_directory_with_trailing_slash_and_same_name_as_command() {
+    use nu_test_support::playground::Playground;
+
+    Playground::setup("cd_test_5_1", |dirs, sandbox| {
+        sandbox.mkdir("cd");
+
+        let actual = nu!(
+            cwd: dirs.test(),
+            r#"
+                cd/
```

Comment:
Forward slash also works in Windows, but it's not `std::path::MAIN_SEPARATOR`. My guess is that some of the `ichwh` and `is_dir` checks in `maybe_autocd` don't care.